app/models/placements/academic_year.rb

```diff
@@ -1,2 +1,24 @@
+# == Schema Information
+#
+# Table name: academic_years
+#
+#  id         :uuid             not null, primary key
+#  ends_on    :date
+#  name       :string
+#  starts_on  :date
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
 class Placements::AcademicYear < AcademicYear
+  def self.current
+    for_date(Date.current)
+  end
+
+  def next
+    Placements::AcademicYear.for_date(starts_on + 1.year)
+  end
+
+  def previous
+    Placements::AcademicYear.for_date(starts_on - 1.year)
```

Comment:
```suggestion
    for_date(starts_on - 1.year)
```

Reply:
@JamieCleare2525 This is an instance method rather than a class method so tests break if `Placements::AcademicYear` is removed.